docs/source/models/other_models/alzheimers_proportional_multistate_life_table/index.rst

```diff
@@ -0,0 +1,309 @@
+    - 
+    - 
+    - 
+    - 
+  * - 61
+    - Male
+    - 
+    - 
+    - 
+    - 
+    - 
+  * - ...
+    - ...
+    - 
+    - 
+    - 
+    - 
+    - 
+  * - 79
+    - Female
+    - 
+    - 
+    - 
+    - 
+    - 
+  * - 79
+    - Male
+    - 
+    - 
+    - 
+    - 
+    - 
+
+.. list-table:: Testing States
+  :header-rows: 1
+
+  * - State
+    - Description
+  * - Untested
+    - MSLT simulants do not receive a BBBM test unless their lifetime :ref:`propensity <bbbm_propensity>`
+      is less than the current time-specific :ref:`test rate <bbbm_rates>`, which is monotonic increasing.
+      We avoid modeling individual propensities by testing a certain fraction of the untested subpopulation
+      on each time step that the test rate increases. The fraction corresponds to the magnitude of the increase.
+  * - Positive
+    - Simulants who test positive remain in this state until they age out. Continuing to track these simulants
+      is important to match the Vivarium simulation, which uses a :ref:`"person-time ever eligible" <alz_observer_outputs>`
+      observer to validate test rates.
+  * - Negative (0 y ago)
+    - Tested simulants move either to the positive state or this state. On the next time step they will move to
+      the "one year ago" state.
+  * - Neg (1 y)
+    - 
+  * - Neg (2 y)
+    - On the next time step simulants will move to either the positive or negative 0 years ago state.
+
+
+
+Initializing the Population
++++++++++++++++++++++++++++
+
+We initialize the susceptible population using the 2021 year forecasted population data :math:`P_{2021}` and the 
+initial simulation "all states" Alzheimer's prevalence :math:`prev_{all}`. Our MSLT initial population is 
+:math:`P_{2021} * (1 - prev_{all})`. This equation represents the non-simulation population at the start of the
+simulation period. 
+
+The only adjustment needed is to modify the age group sizes. The initial population data :math:`P_Y` is from 
+GBD and age groups span five years, eg 60-64 year olds. To simplify our annual timesteps, we divide
+these age groups into single year ages, eg 60 year olds, 61 year olds. We divide the five-year GBD 
+age group populations by five to get single-year age group populations.
+
+Since BBBM testing doesn't begin until 2030, the entire susceptible population is initialized to the 
+untested state.
+
+Model Scale
+-----------
+
+The population counts in the MSLT are at full scale for the susceptible population for the location.
+The MSLT testing results do not need to be scaled.
+
+
+On Time Step
+++++++++++++
+
+On each time step, the MSLT rows and columns and observers are updated to reflect population changes,
+tests and treatments. 
+
+We use an annual time step in order to best reflect our annual test rate targets. 
+
+On each time step, new simulants must be added to the MSLT. The only way to enter the MSLT is to turn 
+60 years old, since it models all 60-80 year-olds who are not in the Vivarium simulation.
```

Comment:
Turn 60 years old and *not* be BBBM-positive for AD;  did you use the `P_{year}*(1-prev_{all})` approach here, like you did above?